data/101/844/515/101844515-alt-quattroshapes.geojson

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 101844515,
+  "type": "Feature",
+  "properties": {
+    "src:geom":"whosonfirst",
```

Comment:
Are these supposed to say `quattroshapes`?

Reply:
Updated to `quattroshapes` in the most recent commit.